src/krm3/utils/dates.py

```diff
@@ -54,6 +54,10 @@ def week_of_year(self) -> int:
     def is_holiday(self) -> bool:
         return not get_country_holidays().is_working_day(self.date)
 
+    @property
+    def is_non_working_day(self) -> bool:
+        return self.is_holiday or self.day_of_week_short in ['Sat', 'Sun']
```

Comment:
It would be statistically more computationally efficient to invert the evaluation order of the expression